packages/newsroom-signup/src/NewsroomProfile/NewsroomBio.tsx

```diff
@@ -31,8 +32,36 @@ export interface NewsroomBioProps extends StepProps {
 
 const LogoFormWrap = styled.div`
   display: flex;
-  justify-content: space-between;
+  justify-content: flex-start;
   margin-top: -4px;
+
+  small {
```

Comment:
This is a dumb fix for conflicting error text + help text, will add a ticket to clean this up